uaclient/util.py

```diff
@@ -839,9 +839,7 @@ def is_installed(package_name: str) -> bool:
         return False
 
 
-def handle_message_operations(
-    msg_ops: List[Union[str, Tuple[Callable, Dict]]],
-) -> bool:
+def handle_message_operations(msg_ops: Optional[MessagingOperations],) -> bool:
```

Comment:
Why are marking the `msg_ops` as `Optional` ?

Reply:
This makes the logic over in `FIPSEntitlement.messaging` a bit easier because we can set `"post_enable": None` if not on a container. And it seemed like a reasonable check to make here.